### DISCUSSION TITLE
[Feature Request] Provide a way to remove the inline styles in the Vimeo player inside the iframe

### DISCUSSION THREAD

**Discussion by ghost** [vimeo/player.js#784] [Ideas]
### Expected Behavior
Remove the inline styles in the element with an id of "player" inside the iframe
![image](https://user-images.githubusercontent.com/64056086/138978515-998b0302-314a-47b3-84bf-85d661657e9f.png)

### Actual Behavior
Currently, it has inline styles for width and height which makes the different videos not have the same dimensions. I have a fixed height and width for the video container, and I just want the Vimeo controls to be in full width, even if the video is not.
![image](https://user-images.githubusercontent.com/64056086/138978504-44d34c0f-585b-4a35-91b9-ea912a2ac760.png)

### Steps to Reproduce
I just copied and pasted this sample Vimeo code to JSFiddle:
https://github.com/vimeo/player.js#pre-existing-player

**Comment by lssamlyon** [comment]
Thanks for reporting this to us.  Let me enquire with the team about this issue and I will update this thread with more details.

**Comment by ghost** [comment]
Thanks @lssamlyon . For clarification, I just need a way to remove the inline styles, not remove them by default.

**Comment by lssamlyon** [comment]
Great. Im going to promote this up to our discussions area for further consideration.

**Comment by robbiecren07** [comment]
Can this be done? I'm embedding a Vimeo video and I want it to be full screen but not with aspect ratio and the inline css for `max-width` and `height` is preventing this.

**Comment by Jayshua** [comment]
I've found that setting the transparent attribute (as a query param to the iframe, or as a property on the Vimeo.Player config object) to false causes these styles to be removed.

```js
		const player = new Vimeo.Player(element, {
			id: "...",
			transparent: false,
		});
```

**Comment by lanelps** [comment]
Would really love a more official solution to this as well so that I can set object-fit: cover on my Vimeo video.
